packages/server/LICENSE.md

```diff
@@ -0,0 +1,19 @@
+# MIT License
+
+Copyright (c) 2024-present [these people](https://github.com/xeho91/svelte-docgen/graphs/contributors)
```

Comment:
nits

```suggestion
Copyright (c) 2024-present [these people](https://github.com/svelte-docgen/svelte-docgen/graphs/contributors)
```

Reply:
Good catch! Thoe files actually were supposed to be symlinks to the root `LICENSE.md`. I fixed it now. I've also changed inside `vite-plugin-svelte-docgen` package.